src/nn/Enums.js

```diff
@@ -1264,6 +1264,42 @@ export const OperationCode = {
    */
   MAXIMUM: 65,
 
+  /** Computes parametric rectified linear activation on the input tensor element-wise.
```

Comment:
@cuiyanx Please align with the explanation of [ANEURALNETWORKS_PRELU](https://developer.android.com/ndk/reference/group/neural-networks#group___neural_networks_1ggaabbe492c60331b13038e39d4207940e0a82e7e7c709ad117a0178203217be30cf), update "**Computes parametric rectified linear activation on the input tensor element-wise.** "with "**_Parametric Rectified Linear Unit._**" 
Also using "**_It follows: f(x) = alpha * x for x < 0, f(x) = x for x >= 0, where alpha is a learned array with the same OperandCode and compatible dimensions as input x._**"